src/db/StockPathHelper.js

```diff
@@ -0,0 +1,71 @@
+import { adultImageNames, kidImageNames } from './imageNames';
+
+function adultImagesPath() {
+  const domain = window.location.href;
+  if (domain.includes('localhost:3000') || domain.includes('sanapolku')) {
+    return 'sanapolku/data/paths/adult/images';
+  }
+  return 'public/data/paths/adult/images'; // Path for testing
```

Comment:
Missä tapauksessa tätä käytetään?

Reply:
Testejä varten koska lokaalisti polku kuviin on eri kun silloin kun sovellus on deployattuna. public/data/paths/adult/images polkua käytetään siis muunmuassa StockPathHelper -> getAdultPath -> should return an array of adult image paths testissä

Reply:
Eihän yksikkötesteissä koskaan haeta mitään mitä tuon polun takaa löytyisi? Eikä siis kuuluisikaan. Tästä funktiosta voi ottaa kokonaan tuon domain tarkastelun pois ja aina palauttaa sanapolku/ alkuinen polku, testitkin toimivat suoraan sillä.

Reply:
Yksikkötesteissä käyettiin tuota ja luulin että se oli tarpeellinen kun joskus aiemmin ei toiminut tuolla sanapolku alkavalla osoitteella. Nyt näemmä toimii